### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Vova-Stoliar**
- [DEMO LINK](https://Vova-Stoliar.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Vova-Stoliar.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Comment by SashaRudenko** [comment]
Можливо, забагато марджинів в блоках.

**Review by SashaRudenko — APPROVED** [review]

File: src/styles/blocks/catalog.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.catalog {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  box-sizing: content-box;
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/catalog.scss:5]:
```suggestion

```
```
* {
 box-sizing: border-box;
}
```

File: src/styles/utilits/_extends.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+%flex-center-between {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+%flex-center {
+  display: flex;
+  align-items: center;
+}
```
Comment (Ihor-Karpyn) [inline src/styles/utilits/_extends.scss:10]: redundant

File: src/styles/utilits/_mixins.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+@mixin image-size($size) {
```
Comment (Ihor-Karpyn) [inline src/styles/utilits/_mixins.scss:2]: square?

File: src/styles/utilits/_mixins.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+@mixin f-size-weight-l-height ($f-size,$f-weight,$l-height) {
+  font-size: $f-size;
+  font-weight: $f-weight;
+  line-height: $l-height;
+}
+
+@mixin f-size-weight ($f-size, $f-weight) {
+  font-size: $f-size;
+  font-weight: $f-weight;
+}
```
Comment (Ihor-Karpyn) [inline src/styles/utilits/_mixins.scss:16]:
```suggestion

```

redundant

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
Almost done, let's fix some problems.

File: src/styles/utilits/_mixins.scss [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
+@mixin image-square($size) {
```
Comment (Ihor-Karpyn) [inline src/styles/utilits/_mixins.scss:2]:
```suggestion
@mixin square($size) {
```

Can i use this mixin with div?  Or its only for img?

**Review by Ihor-Karpyn — APPROVED** [review]
Good job🎉

**Pull request closed without merging**